interface/themes/tabs_style_full.scss

```diff
@@ -120,7 +120,6 @@ body {
 }
 
 .tabControls {
-  background: var(--light);
```

Comment:
hi @ophthal , the removal of this line is causing the tests to fail

Reply:
pretty sure e2e testing is failing because of the removal of the tabNotchosen class above